db/seeds.rb

```diff
@@ -113,25 +148,130 @@
+#  Votes SPWM - agm1  #
+#######################
+
+# First Agenda Approval
+
+# User:Bob
+@first_agenda_vote1 = Vote.create!(agenda_item_id: @business1.id, voting_user_id: @bob.id, vote_type: "In Favour")
+# User:Sylvia
+@first_agenda_vote2 = Vote.create!(agenda_item_id: @business1.id, voting_user_id: @sylvia.id, vote_type: "In Favour")
+# User:Jason
+@first_agenda_vote3 = Vote.create!(agenda_item_id: @business1.id, voting_user_id: @jason.id, vote_type: "In Favour")
+# User:Milly
+@first_agenda_vote4 = Vote.create!(agenda_item_id: @business1.id, voting_user_id: @milly.id, vote_type: "In Favour")
+
+# Pledge Approval
+
+# User:Bob
+@pledge_vote1 = Vote.create!(agenda_item_id: @document1.id, voting_user_id: @bob.id, vote_type: "In Favour")
+# User:Sylvia
+@pledge_vote2 = Vote.create!(agenda_item_id: @document1.id, voting_user_id: @sylvia.id, vote_type: "In Favour")
+# User:Jason
+@pledge_vote3 = Vote.create!(agenda_item_id: @document1.id, voting_user_id: @jason.id, vote_type: "In Favour")
+# User:Milly
+@pledge_vote4 = Vote.create!(agenda_item_id: @document1.id, voting_user_id: @milly.id, vote_type: "In Favour")
+
+# Basement Location Approval
+
+# User:Bob
+@location_vote1 = Vote.create!(agenda_item_id: @action_item1.id, voting_user_id: @bob.id, vote_type: "In Favour")
+# User:Sylvia
+@location_vote2 = Vote.create!(agenda_item_id: @action_item1.id, voting_user_id: @sylvia.id, vote_type: "In Favour")
+# User:Jason
+@location_vote3 = Vote.create!(agenda_item_id: @action_item1.id, voting_user_id: @jason.id, vote_type: "In Favour")
+# User:Milly
+@location_vote4 = Vote.create!(agenda_item_id: @action_item1.id, voting_user_id: @milly.id, vote_type: "In Favour")
+
+# Jason Allowed to Remain in Society
+
+# User:Bob
+@keep_jason_vote1 = Vote.create!(agenda_item_id: @motion1.id, voting_user_id: @bob.id, vote_type: "In Favour")
+# User:Sylvia
+@keep_jason_vote2 = Vote.create!(agenda_item_id: @motion1.id, voting_user_id: @sylvia.id, vote_type: "Opposed")
+# User:Jason
+@keep_jason_vote3 = Vote.create!(agenda_item_id: @motion1.id, voting_user_id: @jason.id, vote_type: "Abstain")
+# User:Milly
+@keep_jason_vote4 = Vote.create!(agenda_item_id: @motion1.id, voting_user_id: @milly.id, vote_type: "In Favour")
 
-# Votes
+# First President Election
+
+# User:Bob
+@first_president_vote1 = Vote.create!(agenda_item_id: @election1_1.id, voting_user_id: @bob.id, vote_type: "In Favour")
+# User:Sylvia
+@first_president_vote2 = Vote.create!(agenda_item_id: @election1_1.id, voting_user_id: @sylvia.id, vote_type: "In Favour")
+# User:Jason
+@first_president_vote3 = Vote.create!(agenda_item_id: @election1_1.id, voting_user_id: @jason.id, vote_type: "In Favour")
+# User:Milly
+@first_president_vote4 = Vote.create!(agenda_item_id: @election1_1.id, voting_user_id: @milly.id, vote_type: "In Favour")
+
+# First Secretary Election
+
+# User:Bob
+@first_secretary_vote1 = Vote.create!(agenda_item_id: @election1_2.id, voting_user_id: @bob.id, vote_type: "In Favour")
```

Comment:
got a nil class error here since `@election1_2.id` should be `@election2_1.id` - otherwise everything seeded perfectly 😻